elements/rh-cta/docs/00-overview.md

```diff
@@ -1,218 +1,31 @@
-  different topics, but they all must be the same style, color, and height.
-
-  {% example palette="lightest",
-             width=684,
-             alt="Call to action component usage, Brick (icon) variant",
-             src="./cta-usage-brick-icon.svg" %}
-
-### Primary (white) {style="margin-block:var(--rh-space-3xl) 0"}
-
-  The Primary (white) variant can be used if the red variant conflicts with other 
-  elements or violates accessibility standards.
-
-
-
-## Best practices
-
-### Changing styles
-
-  Do not modify the styles of any variant for any reason.
-
-  {% example palette="wrong",
-             width=598,
-             alt="Call to action component, best practice 1",
-             src="./cta-bestpractice-1.svg" %}
-
-### Too many links
-
-  Do not group more than three variants together otherwise a user might have trouble making a selection if there are too many link choices.
-
-  {% example palette="wrong",
-             width=635,
-             alt="Call to action component, best practice 2",
-             src="./cta-bestpractice-2.svg" %}
-
-### Using multiple Primary variants
-
-  Do not use multiple Primary variants in the same group, use of that variant is reserved to represent the most important link.
-
-  {% example palette="wrong",
-             width=305,
-             alt="Call to action component, best practice 3",
-             src="./cta-bestpractice-3.svg" %}
-
-### Mixing variant groups
-
-  Do not group more than two variants together anywhere.
-
-  {% example palette="wrong",
-             width=405,
-             alt="Call to action component, best practice 4",
-             src="./cta-bestpractice-4.svg" %}
-
-### Mixing Bricks
-
-  Do not group different Brick variants together anywhere, use one only.
-
-  {% example palette="wrong",
-             width=552,
-             alt="Call to action component, best practice 5",
-             src="./cta-bestpractice-5.svg" %}
-
-### Changing hierarchy
-
-  Do not change the hierarchy when grouping, variants that are higher in hierarchy should go first.
-
-  {% alert state="warning", title="Warning" %}
-    Do not change the hierarchy when grouping unless the change is to accommodate 
-    right-to-left languages.
-  {% endalert %}
-
-  {% example palette="wrong",
-             width=690,
-             alt="Call to action component, best practice 6",
-             src="./cta-bestpractice-6.svg" %}
-
-### Stretching
-
-  Do not stretch any other variants to fit a container or grid, only the Brick variants can stretch.
-
-  {% example palette="wrong",
-             width=664,
-             alt="Call to action component, best practice 7",
-             src="./cta-bestpractice-7.svg" %}
+## When to use
 
+- When you need users to navigate to a different page or domain
+- When you need to arrange links in different arrangements like in a row or grid
+- When you need to hide or reveal content
 
+{% repoStatus type="Element" %}
```

Comment:
@coreyvickery Repo status won't work for CTA until this PR is pushed through: https://github.com/RedHat-UX/red-hat-design-system/pull/1057